### PR TITLE
Sth 0009-Initial feature for the user crud

### PR DESCRIPTION
**Pull request by DilbaniE**
Add a new folder in the structure

### CODE REVIEW

File: stellarhome/src/main/java/com/stellarhome/DTO/UserDTO.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.stellarhome.DTO;
+
+import lombok.*;
+
+@Data
+@Getter
```
Comment (josanchezag) [inline stellarhome/src/main/java/com/stellarhome/DTO/UserDTO.java]: You should looking for a single annotation to merge all those @Setter,@Getter or simple decide to use @Builder @DilbaniE @DilbaniEB

File: stellarhome/src/main/java/com/stellarhome/controller/RollController.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.stellarhome.controller;
+
+import com.stellarhome.endPoints.IUserEndPoint;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RollController {
```
Comment (josanchezag) [inline stellarhome/src/main/java/com/stellarhome/controller/RollController.java]: Rename the class to RoleController @DilbaniE

**Review by josanchezag — COMMENTED** [review]
Some comments

File: stellarhome/src/main/java/com/stellarhome/controller/RollController.java (hunk 2 above)
Comment (DilbaniE) [reply stellarhome/src/main/java/com/stellarhome/controller/RollController.java]: change name to RoleController

**Review by josanchezag — APPROVED** [review]
GLTM

**Pull request merged**